src/domain/entities/disease-outbreak-event/DiseaseOutbreakEvent.ts

```diff
@@ -3,22 +3,34 @@ import { IncidentActionPlan } from "../incident-action-plan/IncidentActionPlan";
 import { IncidentManagementTeam } from "../incident-management-team/IncidentManagementTeam";
 import { TeamMember } from "../incident-management-team/TeamMember";
 import { OrgUnit } from "../OrgUnit";
-import { Id, NamedRef } from "../Ref";
+import { Code, Id, NamedRef } from "../Ref";
 import { RiskAssessment } from "../risk-assessment/RiskAssessment";
 import { Maybe } from "../../../utils/ts-utils";
 import { ValidationError } from "../ValidationError";
 
 export const hazardTypes = [
     "Biological:Human",
     "Biological:Animal",
+    "Biological:HumanAndAnimal",
     "Chemical",
     "Environmental",
     "Unknown",
 ] as const;
 
 export type HazardType = (typeof hazardTypes)[number];
 
-export type IncidentStatusType = "Watch" | "Alert" | "Respond" | "Closed" | "Discarded";
+export enum IncidentStatus {
+    WATCH = "WATCH",
+    ALERT = "ALERT",
+    RESPOND = "RESPOND",
+    CLOSED = "CLOSED",
+    DISCARDED = "DISCARDED",
```

Comment:
[no need to change] When in doubt about the recommended casing, go to the Revealed Truth truth :)

https://www.typescriptlang.org/docs/handbook/enums.html